torchbenchmark/models/opacus_cifar10/__init__.py

```diff
@@ -0,0 +1,81 @@
+import torch
+import torch.optim as optim
+import torch.nn as nn
+import torchvision.models as models
+from opacus.utils.module_modification import convert_batchnorm_modules
+from opacus import PrivacyEngine
+
+from ...util.model import BenchmarkModel
+
+
+class Model(BenchmarkModel):
+    def __init__(self, device=None, jit=False):
+        super().__init__()
+        self.device = device
+        self.jit = jit
+
+        self.model = models.resnet18(num_classes=10)
+        self.model = convert_batchnorm_modules(self.model)
+        self.model = self.model.to(device)
+
+        self.example_inputs = (
+            torch.randn((64, 3, 32, 32), device=self.device),
+        )
+        self.example_target = torch.randint(0, 10, (64,), device=self.device)
+
+    def get_module(self):
+        if self.jit:
+            raise NotImplementedError()
+
+        return self.model, self.example_inputs
+
+    def train(self, niter=3):
```

Comment:
any reason you picked niter=3? 

in practice we've tended to not use niter, since we wrapped it in pytest which does figure out a number of iterations based on how long each takes.

Reply:
Copied it from https://github.com/pytorch/benchmark/blob/3015d62c6c8825521c45cbaf89cc498742bee65b/torchbenchmark/models/resnet18/__init__.py#L23. I'll delete it, having pytest figure out iterations sounds good